selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/NavigateToFile.java

```diff
@@ -124,8 +124,8 @@ public void waitFileNamePopUp() {
    *
    * @param text a text that should be into list
    */
-  public void waitListOfFilesNames(final String text) {
-    new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC)
+  public Boolean waitListOfFilesNames(final String text) {
```

Comment:
"Wait" doesn't require returning the value. IMHO it's better to rename method to `isFilenameSuggested(nameFragment)` according to the actual commands.